kotlin-yaml/src/jsMain/kotlin/yaml/parse.kt

```diff
@@ -0,0 +1,19 @@
+package yaml
+
+@JsModule("yaml")
+external object YAML {
+    /**
+     * Parse an input string into JavaScript.
+     *
+     * Only supports input consisting of a single YAML document; for multi-document
+     * support you should use `YAML.parseAllDocuments`. May throw on error, and may
+     * log warnings using `console.warn`.
+     *
+     * @param str - A string with YAML formatting.
+     * @param reviver - A reviver function, as in `JSON.parse()`
+     * @returns The value will match the type of the root value of the parsed YAML
+     *   document, so Maps become objects, Sequences arrays, and scalars result in
+     *   nulls, booleans, numbers and strings.
+     */
+    fun <T> parse(src: String): T
```

Comment:
Top-level function expected. `YAML` object is redundant

Reply:
My intention was to have it similar to the typical TS usage which is `YAML.parse` similar to `JSON.parse`. But I can of course change it.

Reply:
Intention is valid.
<img width="195" alt="Screenshot 2025-01-20 at 23 25 29" src="https://github.com/user-attachments/assets/b5a41ecb-f40c-414f-9ad2-594449f9c374" />
In both cases `parse` looks like "common" name. Other names are more unique and don't need namespace.
If it will be a problem we will use more specific names.
In case of `yaml` it's also bundle size problem :(

Reply:
You can also do `yaml.parse(...)` instead of an import, so maybe that is also good enough if it is more consistent for the wrappers codebase.

What do you mean by bundle size?
Is the bundle bigger if you use an object namespace?
Or what did you mean?

Reply:
> Is the bundle bigger if you use an object namespace?

Yes, all object will be included

Reply:
[Done](https://github.com/JetBrains/kotlin-wrappers/commit/2ed2b48897c283188ffbd703aa2b061e78fdc945)

Reply:
Oh, I didn't even mean in kotlin wrappers, but in TS or JS. :-D
But yeah, within this codebase it should probably also be consistent and that's more important. :-)
Now I just wonder that you didn't pull out the other JSON functions. :-D

Reply:
Oh, you did. :-D